test/pytorch_test_base.py

```diff
@@ -27,6 +27,8 @@
         'test_min_max_nan',  # XLA min/max ignores Nans.
         'test_min_max_binary_op_nan',  # XLA min/max ignores Nans.
         'test_copy_broadcast',
+        'test_advancedindex_mixed_cpu_devices_xla',  # TODO @wonjoo fails with functionalization
```

Comment:
For tests being skipped here, it would be good to have corresponding GH issues to suggest why they fail. If it's an upstream problem, it can just be the upstream GH issue.

Reply:
Sounds good, I'll create an issue on our end for each one.

Reply:
Created an issue for each test tagged with "LTC".